internal/pkg/postgres/postgres.go

```diff
@@ -92,6 +92,50 @@ func (c *Client) CreateAlert(ctx context.Context, smeeAlert smee.Alert) (smee.Is
 		return smee.Issue{}, fmt.Errorf("unable to commit transaction: %w", err)
 	}
 
+	if err := c.unacknowledgeIssue(ctx, tx, issID); err != nil {
+		return smee.Issue{}, fmt.Errorf("unable to unaknowledge issue: %w", err)
+	}
+
+	return smeeIss, nil
+}
+
+func (c *Client) AcknowledgeAlertsForIssue(ctx context.Context, issueID string) (smee.Issue, error) {
```

Comment:
Let's change the name of this function to something like `AcknowledgeIssue` so that it's more clear what's going on. I was confused originally reading through the code about where we were actually acknowledging the issue.